coxeter/families/common.py

```diff
@@ -117,23 +119,25 @@ def make_vertices(cls, n):
 JohnsonFamily = TabulatedGSDShapeFamily.from_json_file(
     os.path.join(_DATA_FOLDER, "johnson.json"),
     classname="JohnsonFamily",
-    docstring="""The family of Johnson solids (92 total).
+    docstring="""The family of Johnson solids, as enumerated in \
+    :cite:`Johnson1966` (92 total).
+
 The following parameters are required by this class:
     - name: The name of the JohnsonFamily solid. A full list is available in \
-            10.1126/science.1220869: :cite:`Damasceno2012`. In general, shape names \
+            :cite:`Johnson1966`. In general, shape names \
             should have the first character of each word capitalized, with spaces \
             between words (e.g. "Elongated Triangular Cupola"). Pyramids and \
             dipyramids are named from their base polygon (e.g. "Square Pyramid" \
             or "Elongated Pentagonal Dipyramid").
 """,
 )
 
-
 PyramidDipyramidFamily = TabulatedGSDShapeFamily.from_json_file(
     os.path.join(_DATA_FOLDER, "pyramid_dipyramid.json"),
     classname="PyramidDipyramidFamily",
     docstring="""The family of regular equilateral pyramids and dipyramids (6 total).
-The following parameters are required by this class:
+
+    The following parameters are required by this class:
```

Comment:
This doesn't render properly. I think you want the following:
```suggestion
    The following parameters are required by this class:
    
```